actions/ql/lib/codeql/actions/Ast.qll

```diff
@@ -242,6 +242,8 @@ class Step extends AstNode instanceof StepImpl {
 
   If getIf() { result = super.getIf() }
 
+  AstNode getUses() { result = super.getUses() }
```

Comment:
Isn't there a more specific type for uses-based steps?

Reply:
No you are right, there is a `UsesStep`.